index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- fontawesomecdn -->
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+    <!-- link to css file -->
+    <link rel="stylesheet" href="./style.css">
+    <title>Lynette || My Portfolio</title>
+</head>
+<body>
+    <header>
+
+        
+        <div class="container">
+       <div class="wrapper">
+                <nav>
+                    <a href="#" class="welcome">Welcome</a>
+                    <a class="burger"><i class="fas fa-bars"></i></a>
+                    <ul class="nav-list">
+                        <li><a class="nav-item" href="#">Portfolio</a></li>
+                        <li><a class="nav-item" href="#">About</a></li>
+                        <li><a class="nav-item" href="#">Contact</a></li>
+                    </ul>
+
+                </nav>
+            </div>
+        </div>
+    </header>
+
+
+    <section class="intro">
+        <div class="content">
+            <h1 class="greetings">Hey there. I'm Lynette</h1>
+            <h2 class="first">  I'm a software developer</h2>
+            <p class="second"> I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hesitate to contact me.
+            </p>
+        
+
+            <ul class="page-links">
+                <li>
+                    <a class="icon-list" href="https://github.com/iLynette"><i class="fab fa-github"></i></a>
+                </li>
+
+                <li>
+                    <a class="icon-list" href="https://www.linkedin.com/in/lynette-acholah/"><i class="fab fa-linkedin"></i></a>
+                </li>
+
+                <li>
+                    <a class="icon-list" href="https://github.com/iLynette"><i class="fab fa-angellist"></i></a>
+                </li>
+
+                <li>
+                    <a class="icon-list" href="https://twitter.com/acholah_lynette"><i class="fab fa-twitter"></i></a>
+                </li>
+
+                <li>
+                    <a class="icon-list" href="https://medium.com/@acholahlynette111"><i class="fab fa-medium"></i></a>
+                </li>
+            </ul>
+        </div>
+
+
+    </section>
+</body>
```

Comment:
- [ ][OPTIONAL] I think your project is missing the background images, fonts and font sizes specified in your chosen template. I suggest you follow the specifications in your chosen design.
- [ ][OPTIONAL] I also noticed that your web is overflowing horizontally. I suggest you try to fix that too.

Your Design | Figma Design |
:---------------:|:-------------------:|
![d1](https://user-images.githubusercontent.com/39523312/148514438-6c3d1780-608c-4760-808b-68dfe6a6fe3a.png) |![d2](https://user-images.githubusercontent.com/39523312/148514443-2461d1ee-b6e8-4acd-9567-332ffdfa2fc5.png)